addons/gatekeeper/3.0.x/gatekeeper-1.yaml

```diff
@@ -29,7 +29,7 @@ spec:
   chartReference:
     chart: gatekeeper
     repo: https://mesosphere.github.io/charts/staging
-    version: 0.1.0
+    version: 0.3.3
```

Comment:
Does this bump the version of gatekeeper? See line 11

Reply:
It bumps the gatekeeper chart, which has been updated but never bumped here.